src/main/include/RobotContainer.h

```diff
@@ -6,22 +6,25 @@
 
 #include <frc2/command/CommandPtr.h>
 
-#include "subsystems/Intake.h"
 #include "T34CommandXboxController.h"
-#include "subsystems/SwerveDrive.h"
 #include "commands/ControllerDriveCommand.h"
+#include "subsystems/Climber.h"
+#include "subsystems/Intake.h"
+#include "subsystems/SwerveDrive.h"
 
-class RobotContainer {
- public:
+class RobotContainer
+{
+public:
   RobotContainer();
 
-  static RobotContainer* Get();
   std::shared_ptr<t34::SwerveDrive> swerve_drive;
   std::shared_ptr<t34::T34CommandXboxController> ctrl;
   t34::Intake intake;
```

Comment:
Should be private. Also naming convention:

```suggestion
  t34::Intake m_intake;
```

Reply:
Has been fixed